packages/vscode-extension/src/commands/Deploy.ts

```diff
@@ -0,0 +1,44 @@
+import SuiteCloudRunner from '../core/SuiteCloudRunner';
+import CommandsMetadataSingleton from '../service/CommandsMetadataSingleton';
+import { getRootProjectFolder, unwrapExceptionMessage } from '../util/ExtensionUtil';
+import { scloudOutput } from '../extension';
+import { MessageService } from '../service/MessageService';
+
+
+export default async function deploy() {
+
+    const deployInfo = new MessageService('deploy');
```

Comment:
i'd give it a more meaningful name, like `const bubbleService = new BubbleService('deploy');` or something that explicitly says it's not a logging mechanism

Reply:
I was just following the vscode api naming.